include/click/dpdkdevice.hh

```diff
@@ -1,6 +1,23 @@
 #ifndef CLICK_DPDKDEVICE_HH
 #define CLICK_DPDKDEVICE_HH
 
+//Prevent bug under some configurations (like travis-ci's one) where these macros get undefined
+#ifndef UINT8_MAX
```

Comment:
Is it possible that we're not #including <stdint.h> where these macros should be defined?